test/monitoring_console/monitoring_console_test.go

```diff
@@ -501,7 +500,7 @@ var _ = Describe("Monitoring Console test", func() {
 			testenv.SearchHeadClusterReady(deployment, testenvInstance)
 
 			// Wait for MC to go to Updating Phase
```

Comment:
Nit: If we now wait for Pending instead of Updating, would be more clear to edit that comment above too